src/it/scala/org/constellation/ClusterHealthCheckTest.scala

```diff
@@ -0,0 +1,98 @@
+package org.constellation
+
+import akka.actor.ActorSystem
+import akka.stream.ActorMaterializer
+import akka.testkit.TestKit
+import org.constellation.crypto.KeyUtils
+import org.constellation.util.{APIClient, HealthChecker, Metrics}
+import org.scalatest.{BeforeAndAfterAll, FlatSpecLike}
+
+import scala.concurrent.ExecutionContextExecutor
+import scala.concurrent.duration.{FiniteDuration, _}
+
+/**
+  *
+  */
+class ClusterHealthCheckTest
+    extends TestKit(ActorSystem("ClusterHealthCheckTest"))
+    with FlatSpecLike
+    with BeforeAndAfterAll {
+
+  override def afterAll {
+    TestKit.shutdownActorSystem(system)
+  }
+
+  implicit val materialize: ActorMaterializer = ActorMaterializer()
+  implicit val executionContext: ExecutionContextExecutor = system.dispatcher
+
+  val (ignoreIPs, auxAPIs) = ComputeTestUtil.getAuxiliaryNodes()
+  val apis: Seq[APIClient] = ComputeTestUtil.createApisFromIpFile(
+    System.getenv().getOrDefault("HOSTS_FILE", "hosts-2.txt"),
+    ignoreIPs
+  )
+
+  // For fixing some old bug, revisit later if necessary
```

Comment:
some old bug? 🤔

Reply:
Once I will be able to complete test fully, I will remove it to check if it's still relevant